spec/features/logout_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+require 'rails_helper'
+
+feature "log out" do
+  scenario "user logs out" do
+    visit register_path
+    set_omniauth(opts = {})
```

Comment:
I think we can create a user, set the session user_id to the users id and than log out without registering?